docker/sbtc/Dockerfile

```diff
@@ -28,6 +28,9 @@ RUN make install-pnpm && make build
 
 # Build the codebase from the local sources.
 # -------------------------------------------
+# We first remove the existing sources so that the there is no conflict
```

Comment:
I guess I'm pondering over if cloning and building `main` from git (above) really gives us anything, over just using `./`? If not, then we could remove these steps altogether..

Reply:
If we keep the `git clone`, it would be good to update the URL to the new org while we're anyways in here.